### DISCUSSION TITLE
turbo-ignore preventing deployments on Vercel via CMS

### DISCUSSION THREAD

**Discussion by ivoilic** [vercel/turborepo#3200] [Help] [answered]
Hi,
I'm using Dato CMS and Turbo for a NextJS project hosted on Vercel.
I noticed all the builds triggered by Dato were being cancelled because of turbo-ignore.
Not sure if this is a Vercel issue or a Turbo issue.

**Comment by anthonyshew** [comment, marked as answer]
Turborepo and `turbo-ignore` are working as intended here.

`turbo` isn't aware of your external content changes. As long as your source code and the information in `turbo.json` are unchanged, your build will hit cache.
